src/libsystemd-network/test-dhcp-client.c

```diff
@@ -102,8 +102,8 @@ static void test_request_basic(sd_event *e) {
 
         /* RFC7844: option 33 (SD_DHCP_OPTION_STATIC_ROUTE) is set in the
          * default PRL when using Anonymize, so it is changed to other option
-         * that is not set by default, to check that it succed setting it.
-         * Ooptions not set by default (using or not anonymize) are option 17
+         * that is not set by default, to check it succeeded setting it.
```

Comment:
Well, both the original and the replacement are not valid English ;(
Maybe "..., to check that it was set successfully." ?

Reply:
fixed.